src/scheduler.py

```diff
@@ -381,33 +397,38 @@ def _run(self, sub_id):
             event = None
             try:
                 event = self._api_helper.receive_event_data(sub_id, block=False)
+                if not event:
+                    # If we received a keep-alive event, just continue
+                    continue
             except Exception as e:
                 self.log.error(f"Error receiving event: {e}, re-subscribing in 10 seconds")
-                time.sleep(10)
-                sub_id = self._api.subscribe('node')
-                subscribe_retries += 1
-                if subscribe_retries > 3:
-                    self.log.error("Failed to re-subscribe to node events")
-                    return False
-                continue
-            if not event:
-                # If we received a keep-alive event, just continue
+                # time.sleep(10)
```

Comment:
why leaving this commented code ? there is no explanation about this.

Reply:
See https://github.com/kernelci/kernelci-pipeline/pull/1264#issuecomment-3132899550 for reference.